docs/calico-with-docker/docker-network-plugin/README.md

```diff
@@ -235,7 +235,27 @@ fail.
 To see the list of networks, use
 
     docker network ls
-        
+
+## Assign Static IP when Starting a Container
+
+With the release of Docker 1.10, support has been added to allow users to
+configure a specific IP address when creating a container.  In order to use
+this feature, Docker requires that you specify the `--subnet` flag when running
```

Comment:
I believe a flag indicates a true/false option, so really --subnet is not actually a flag.